.travis.yml

```diff
@@ -1,9 +1,13 @@
 language: node_js
 node_js: ["0.10"]
+cache:
+  directories:
+    - node_modules
+    - bower
 before_install:
   - export CHROME_BIN=chromium-browser
   - export DISPLAY=:99.0
   - sh -e /etc/init.d/xvfb start
 before_script:
-  - npm install -g bower gulp
```

Comment:
I modified the test script here to use `./node_modules/.bin/gulp` on [[LOC](https://github.com/CrossLead/angular-zendesk-widget/pull/5/files#diff-354f30a63fb0907d4ad57269548329e3R13)]. Did you mean somewhere else?